### PR TITLE
Store the user's username to user in sudo

### PR DESCRIPTION
**Pull request by jonstorer**
Review!

### CODE REVIEW

**Comment by michaeldauria** [comment]
I would make sure that this works first...

**Comment by quentindecock** [comment]
Yeah make sure that it works.

Otherwise looks good.

File: make.sh [hunk 1]
```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 
 echo "Crowdtap Environment Setup Script"
+  WHOAMI="`whoami`"
 
 echo "Checking for SSH key, generating one if it doesn't exist ..."
-  [[ -f ~/.ssh/id_rsa.pub ]] || ssh-keygen -t rsa
+  [[ -f ~/.ssh/id_rsa.pub ]] || ssh-keygen -t rsa && echo "key generated"
```
Comment (jonstorer) [inline make.sh:7]: for some reason, when the SHH key is missing this halts execution

**Comment by jonstorer** [comment]
closed

**Pull request closed without merging**